app_feup/assets/text/TermsAndConditions.txt

```diff
@@ -1,40 +1,53 @@
-Terms and conditions
+# App desenvolvida pelo NIAEFEUP.
```

Comment:
I think it makes sense to change the file extension to `.md` since now it pertains to one